tests/utils.py

```diff
@@ -19,7 +23,45 @@ def unorder(value):
     return unorder(xmltodict.parse(xml))
 
 
+NAMESPACE_ATTRIBUTE = re.compile(' xmlns="[^"]+"', flags=re.UNICODE)
```

Comment:
As oozie workflow xml [supports](https://issues.apache.org/jira/browse/OOZIE-2400) xml namespaces with prefix, this regex doesn't handle that -

```python
In [9]: patt = re.compile(' xmlns="[^"]+"', flags=re.UNICODE)

In [10]: xml_string = """
    ...: <workflow-app xmlns:h="uri:oozie:workflow:0.5" name="descriptive-name">
    ...:     <parameters>
    ...:         <property>
    ...:             <name>property_key</name>
    ...:             <value>property_value</value>
    ...:         </property>
    ...:         <property>
    ...:             <name>other_property_key</name>
    ...:             <value>🤣</value>
    ...:         </property>
    ...:     </parameters>
    ...:     <global>
    ...:         <job-tracker>job-tracker</job-tracker>
    ...:         <name-node>name-node</name-node>
    ...:     </global>
    ...:     <start to="end" />
    ...:     <end name="end" />
    ...: </workflow-app>
    ...: """.strip()

In [11]: patt.sub(str(''), xml_string.strip())
Out[11]: '<workflow-app xmlns:h="uri:oozie:workflow:0.5" name="descriptive-name">\n    <parameters>\n        <property>\n            <name>property_key</name>\n            <value>property_value</value>\n        </property>\n        <property>\n            <name>other_property_key</name>\n            <value>\xf0\x9f\xa4\xa3</value>\n        </property>\n    </parameters>\n    <global>\n        <job-tracker>job-tracker</job-tracker>\n        <name-node>name-node</name-node>\n    </global>\n    <start to="end" />\n    <end name="end" />\n</workflow-app>'

```

Not sure how it affects this PR though